### DISCUSSION TITLE
Libcjose dependency (v2.4.10+)

### DISCUSSION THREAD

**Discussion by trumble117** [OpenIDC/mod_auth_openidc#927] [Q&A] [answered]
Hi, I am working through some system requirements for a new deployment. I have been working with this plugin for a while, predating the release of v2.4.10. Prior to release v2.4.10, cjose >= 0.5.1 is listed as a dependency in the release notes, and I have been carrying this dependency forward without second guessing. I'm looking to deploy the latest release (v2.4.11.3 as of writing) and am seeking clarification regarding if the cjose package is still required for proper operation or not?

**Comment by zandbelt** [comment, marked as answer]
cjose is still required indeed but these days it is more of a standardized dependency that you can get from distribution package repositories, that is why it is no longer explicitly listed (similar to libcurl, libssl, libpcre etc.); if you want the latest and greatest you can get it at https://github.com/zmartzone/cjose/releases

**Reply by trumble117** [reply]
Thank you for the clarification and for the quick reply!
